ui_framework/components/global_styles/variables/_typography.scss

```diff
@@ -0,0 +1,73 @@
+// Font
+
+// Families
+$kuiFontFamily: "Roboto", Helvetica, Arial, sans-serif;
+$kuiCodeFontFamily: "Roboto Mono", monospace;
+
+// Font sizes
+$kuiFontSize:       $kuiSize;
+
+$kuiFontSizeXS:     12px;
+$kuiFontSizeS:      14px;
+$kuiFontSizeM:      $kuiFontSize;
+$kuiFontSizeL:      24px;
+$kuiFontSizeXL:     32px;
+$kuiFontSizeXXL:    48px;
```

Comment:
I'd like to still try and account for older versions of IE since we target enterprise. the `fontSize` mixin basically spits out both, but I need a base pixel value to output.